package.json

```diff
@@ -58,7 +58,7 @@
     "eslint-plugin-prettier": "^3.3.1",
     "express": "^4.17.1",
     "husky": "^5.1.3",
-    "indy-sdk": "^1.16.0",
+    "indy-sdk": "^1.16.0-dev-1633",
```

Comment:
Can I ask what the impact of this upgrade is?

Reply:
Sure. The impact should be minimal. This new version of indy-sdk supports Node 12+ and drops support for NodeJS 8. Node 8 is already end of life for a while and we have never supported it. So in the context of AFJ it just means we support newer NodeJS versions.

However as developers install the indy-sdk themselves they are not required to update to this new version. If they want to use an older version of indy-sdk they can do so. But I don't think there should be any reason not to update. There's nothing functionally different in this indy-sdk version

I think it is nice we can use the Current and LTS NodeJS versions again.

Reply:
Awesome, fully agree with the thought processes here. Thanks for the clarification.